### ISSUE TITLE
Uncaught TypeError: Craft.Formie is not a constructor v1.6.37

### ISSUE THREAD

**Issue by iainsaxon** [verbb/formie#1763]
### Describe the bug

When opening a form for editing in the control panel using Craft 3.9.11 and Formie 1.6.37 I do not see any form editors or functioning elements. If I look at the console I see:

```
122870:849 Uncaught TypeError: Craft.Formie is not a constructor
    at 122870:849:1
(anonymous)	@	122870:849
```

The JS at this target is:

```
<script src="https://test.localhost/cpresources/85c21caf/js/formie-cp.js?v=1710138675"></script>
<script>Craft.cp.setJobInfo([{"id":"141208","delay":0,"status":4,"progress":0,"progressLabel":null,"description":"Sending form notification.","error":"Failed to send notification email: {\"error\":\"Notification email \u201cUser Notification\u201d could not be sent for submission \u201c126268\u201d. The mailer service failed to send the notification: \u201c\u201d.\"}."},{"id":"141232","delay":0,"status":4,"progress":0,"progressLabel":null,"description":"Sending form notification.","error":"Failed to send notification email: {\"error\":\"Notification email \u201cUser Notification\u201d could not be sent for submission \u201c126280\u201d. The mailer service failed to send the notification: \u201c\u201d.\"}."}], false);
new Craft.Formie({
        isStencil: 0,
...
        maxFormHandleLength: 56,
        maxFieldHandleLength: 38,
    });
```

![Screenshot 2024-03-11 at 6 13 48 pm](https://github.com/verbb/formie/assets/546517/c490c3ce-69f0-4a49-a273-6567deb3ab6a)

To sanity check I looked at another site I'd recently performed a CMS update on that was working and found that it was only running Formie v1.6.36.1. If I rolled back to this specify version the Formie control panel JS works again.

### Steps to reproduce

1. Update to Formie 1.6.37
2. Open a form edit screen and see error
3. Composer update to the previous version
4. Open a form edit screen and it should be working

### Form settings

- Multi-page form: No
- Submission Method: Ajax
- Client-side Validation: Yes
- Custom Form Templates: No

### Craft CMS version

3.9.11

### Plugin version

1.67

### Multi-site?

No

**Comment by Tim-Wils** [comment]
I have the same bug

**Comment by engram-design** [comment]
Fixed in [1.6.38](https://github.com/verbb/formie/releases/tag/1.6.38)
